lib/core/src/lib/pipes/date.pipe.ts

```diff
@@ -0,0 +1,30 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Pipe, PipeTransform } from '@angular/core';
+import { isValid } from 'date-fns';
+import { DateFormatTranslationService } from '../form/services/date-format-translation.service';
+
+@Pipe({ name: 'adfDate' })
+export class DatePipe implements PipeTransform {
```

Comment:
the naming of class is going to clash with Angular DatePipe: https://angular.io/api/common/DatePipe
please use AdfDatePipe as the pipe name suggests

Reply:
renamed